src/pages/teachers/lessons/[lessonSlug]/downloads.tsx

```diff
@@ -21,9 +20,9 @@ export type LessonDownloadsCanonicalPageProps = {
   curriculumData: LessonDownloadsCanonical;
 };
 
-const LessonDownloadsCanonicalPage: NextPage<
-  LessonDownloadsCanonicalPageProps
-> = (props) => {
```

Comment:
This type wasn't doing any work and I needed to remove it to make the component fit the types for `withPageAuthRequired`